### PR TITLE
chore(vsc): add experiment for opening folder in new window

### PR DESCRIPTION
**Pull request by tecton**
Add A/B test experiment to open folder in new VS Code window after creation.
It helps to reduce the percentage of lost telemetry events caused by VS Code reload.

Effect:
image

### CODE REVIEW

File: packages/vscode-extension/src/controls/webviewPanel.ts [hunk 1]
```diff
@@ -173,12 +173,7 @@ export class WebviewPanel {
     if (res.isOk()) {
       props[TelemetryProperty.Success] = TelemetrySuccess.Yes;
       ExtTelemetry.sendTelemetryEvent(TelemetryEvent.DownloadSample, props);
-      await globalStateUpdate(GlobalKey.OpenSampleReadMe, true);
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/src/controls/webviewPanel.ts:176]: Will GlobalKey.OpenSampleReadMe updated after this change?

Comment (tecton) [reply packages/vscode-extension/src/controls/webviewPanel.ts:176]: Yes, it's updated in handlers.ts:L518

**Comment by HuihuiWu-Microsoft** [comment]
Do we also consider reloadWindow after migration(Although it should be smaller number compared with create project)?

**Comment by tecton** [comment]
> Do we also consider reloadWindow after migration(Although it should be smaller number compared with create project)?

No, we don't need to change it because:
1. Only 2 migration events happened in 4.x and telemetry data were not lost.
2. It makes sense to reload in current window because it's the same project.

**Review by HuihuiWu-Microsoft — DISMISSED** [review]

**Review by jayzhang — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Pull request merged**